js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -348,7 +348,27 @@ export class MiniAppBridge {
       return this.executor.exec(
         'sendMessageToContactId',
         { contactId: id, messageToContact: message },
-        messageId => resolve(messageId),
+        contactId => resolve(contactId),
+        error => reject(error)
+      );
+    });
+  }
+
+  /**
+   * @param message The message to send to contact.
+   * @returns Promise resolves with an array of contact id which were sent the message.
+   * Can also resolve with an empty array in the case that the message was not sent to any contacts, such as if the user cancelled sending the message.
```

Comment:
Looks like, we could say: "empty (undefined) response" instead of "empty array" as we are planning to return null from mobile sdk side when user cancelled the operation.

Reply:
Return null is our sdk behavior. There is the probability that user can return an empty array.

Reply:
I read the comment on android PR now. Let me update this js sdk doc.